gulpfile.js

```diff
@@ -140,4 +140,9 @@ createTask('unit', 'unit', 'unit');
 createTask('update-packages', 'updatePackages', 'update-packages');
 createTask('validator', 'validator', 'validator');
 createTask('validator-webui', 'validatorWebui', 'validator');
+createTask(
+  'check-video-interface-list',
+  'checkVideoInterfaceList',
+  'check-video-interface-list'
+);
```

Comment:
Nit: For easy readability of this file, put this on one line with `// prettier-ignore` at the end.

Reply:
Or—since it seems to be a property of the listing already—why not default the filename to the task name?

```js
function createTask(taskName, taskFuncName, opt_taskSourceFileName) {
  const taskSourceFileName = opt_taskSourceFileName || taskName;
}
// ...
createTask('check-video-interface-list', 'checkVideoInterfaceList');
```

Reply:
They aren't always the same. This file is going to go away soon, so feel free to disregard this.